### DISCUSSION TITLE
Support for Windows 7

### DISCUSSION THREAD

**Discussion by softboy99** [unoplatform/uno#10435] [Q&A]
## What would you like to be added:
win7 support
## Why is this needed:
there are still many win7 pc all over the world.
## For which Platform:
- [ ] Windows

**Comment by MartinZikmund** [comment]
Windows 7 devices should be supported via the Skia.WPF target, have you tested it?

**Reply by lavahasif** [reply]
i just built normal wpf application in using .net 6 .it work in self contained  publish .but not worked  framework dependent.in uno i frame work dependent publish not working . i couldnt build self contained publish showing error.any solution?

**Reply by jeromelaban** [reply]
Thanks. It's best to avoid opening two discussions for the same topic. Let's [continue here](https://github.com/unoplatform/uno/discussions/10380#discussioncomment-4297401).
